.github/workflows/build.yml

```
@@ -18,7 +18,7 @@ on:
 
 env:
   DOTNET_VERSION: ${{ '8.0.201' }}
-  ENABLE_DIAGNOSTICS: true
+  ENABLE_DIAGNOSTICS: ${{ github.ACTIONS_RUNNER_DEBUG }}
```

Can we add a comment with info about how to turn this on from the Actions/PR on GitHub?

---

Anyone who has access to run a workflow can enable runner diagnostic logging and step debug logging for a workflow re-run. It can also be enabled by setting secrets or variables in the repository containing the workflow. 

This guidance is documented by GitHub [here](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging) and is subject to change. I'll add this and the link as a code comment for reference. 

```suggestion
  # Debug logging enabled via GitHub UI during reruns or set as repo secret.
  # See also https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging
  ENABLE_DIAGNOSTICS: ${{ github.ACTIONS_RUNNER_DEBUG }}
```